packages/components/src/components/icon/icon.jsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import PropTypes  from 'prop-types';
+import React      from 'react';
+
+const getUrlBase = (path = '') => {
```

Comment:
@kelly-binary can you please add a TODO here, this needs to go shared.

Reply:
Please use the TODO tag `[move-to-shared]` as we use that in some other places in the code too.